### ISSUE TITLE
Barako can be cheesed by drowning him in a pit where Barakoaya can’t reach him

### ISSUE THREAD

**Issue by BobMowzie** [BobMowzie/MowziesMobs#541]

**Comment by Darkmega18** [comment]
time to give him a solar magic circle thing that he can sit on perhaps. 
stops him from falling into holes when explosives etc are deployed. making it detrimental to the player to explode or deform the terrain around him as he'll just hover there and not care.

**Comment by BobMowzie** [comment]
This wouldn’t help. Players could construct the pit around his floating seat and drown him in place.
It’s fine though; Barako aggros on players who break his throne or place tnt, so it’s not as though you can do this without a fight.

**Comment by Darkmega18** [comment]
thats why he'd need to float on the water too even if you pour a waterfall on him. right?

~~alternatively water breathing also exists.~~

**Comment by BobMowzie** [comment]
Right, but that also wouldn’t fix it. You could build a roof at water level and prevent him from surfacing.
Logically it doesn’t seem right to give him water breathing.

**Comment by Darkmega18** [comment]
evaporate the water when he super novas/fire explodes. :3

**Comment by Darkmega18** [comment]
He will summon his bois and get shielded and healed tho. so that can suffice.
then that means you've given him a mortal weakness then that you'll just have to hope people don't cheese him with. _[shrug]_
Grief smeif. it's only water. if he's in water, it's likely he's been put there by some idiot combat avoider/big thinker and best you can do is just make sure his healers can shield him quick enough.
_(they usually can in my game cause I made him champion 3 to survive weak attempts at his life. but yeah...)_

**Comment by BobMowzie** [comment]
It’s difficult enough to pull off in recent versions that I’m not worried about it.
